Eatery Watch App Extension/UI/Eatery/CampusEateryMealsView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CampusEateryMealsView.swift
+//  Eatery Watch App Extension
+//
+//  Created by William Ma on 1/7/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import SwiftUI
+
+/// A list of menus that
+struct CampusEateryMealsView: View {
```

Comment:
nit: add new line at the beginning and closing of `struct`
same for the other structs in the PR